### PR TITLE
Add Gazebo Support

### PR DESCRIPTION
**Pull request by civerachb-cpr**
Add support for controlling the PTU inside of a Gazebo simulation.

The behaviour of the simulation isn't 100% identical, but the main ROS interfaces (`/joint_states` and `/ptu/cmd`) are replicated.  A pair of `velocity_controllers/JointPositionController`s are used to control the pan and tilt actuators, with a simple Python node acting as a middle layer between the PID controllers' `.../command` topics and the Flir PTU's `.../cmd` topic.

The biggest difference between the simulation and the real thing is that in the simulation the velocity of the rotation cannot be controlled; the speed of rotation is governed wholly by the PID controllers implemented by the `JointPositionController` instances.  But the angles can be set using the same ROS interface, and the PTU does move, which is the most important aspect.

### CODE REVIEW

**Comment by civerachb-cpr** [comment]
Removed WIP tag. I think this is ready for review.

File: flir_ptu_viz/CMakeLists.txt [hunk 1]
```diff
@@ -4,10 +4,5 @@ project(flir_ptu_viz)
 find_package(catkin REQUIRED COMPONENTS roslaunch)
 catkin_package()
 
-roslaunch_add_file_check(launch/view_model.launch
```
Comment (tonybaltovski) [inline flir_ptu_viz/CMakeLists.txt:7]: Why was this removed?

Comment (civerachb-cpr) [reply flir_ptu_viz/CMakeLists.txt:7]: Re-added. The tests were failing previously and I'd just deleted them while I worked on other stuff and forgot to re-add them.

**Review by tonybaltovski — APPROVED** [review]
